remote_attestation/rust/src/attestation.rs

```diff
@@ -273,3 +580,12 @@ fn verify_attestation(
         Err(anyhow!("Incorrect TEE measurement"))
     }
 }
+
+// TODO(#2273): Use raw bytes for messages since Protobuf is not deterministic.
```

Comment:
Not only that, but I do think the protocol should operate on bytes anyways; i.e. it should return the exact bytes to send over the wire to the other side, not a proto message. Since the caller would not do anything with those proto bytes apart from converting them to bytes immediately anyways. Anyways, let's discuss that separately on the dedicated issue.

Reply:
It also depends on the SC protocol being fleshed out.
Because currently it doesn't specify all the fields exactly (i.e. all fields related to TEE are still vague).